README.md

```diff
@@ -5,32 +5,30 @@
 
```

Comment:
I'd rather leave it towards the bottom, because it won't be useful most of the time. I'm not sure changing vm.max_map_count is needed for every machine. Someone doesn't need to install Swagger if they just want to run local Data Explorer.